contracts/src/security/reentrancy_guard/mod.rs

```diff
@@ -37,7 +37,7 @@ pub const STORAGE_KEY: u32 = openbrush::storage_unique_key!(Data);
 #[openbrush::upgradeable_storage(STORAGE_KEY)]
 #[derive(Default, Debug)]
 pub struct Data {
-    pub status: u8,
+    pub status: Lazy<u8>,
     pub _reserved: Option<()>,
```

Comment:
What's the point of keeping `_reserved`?

Reply:
What could make sense is wrap whole `Data` with `Lazy` -- then the `_reserved` would maybe be justified. But this version is just strange, because `_reserved` will end up in a random place in the main storage, and `status` will be at a different key.

Reply:
If we wrap `Data` with `Lazy` then I don't think you need `_reserved` at all. Otherwise, I agree.